qiskit/providers/models/backendconfiguration.py

```diff
@@ -300,3 +340,115 @@ def __init__(self, backend_name, backend_version, n_qubits, basis_gates,
                          meas_lo_range=meas_lo_range, dt=dt, dtm=dtm,
                          rep_times=rep_times, meas_kernels=meas_kernels,
                          discriminators=discriminators, **kwargs)
+
+    @property
+    def dt(self) -> float:  # pylint: disable=invalid-name
+        """Time delta between samples on the signal channels in seconds."""
+        warnings.warn("The time delta is now returned in units of [s] rather than [ns].")
```

Comment:
This is worth looking at closely: this is a hard change for the user. Is there a better way to do this? We do consider Pulse to sort of be in beta, but can we make this change easier for them? Same for `dtm`. No other existing behavior changes in this PR.

Reply:
I think we may have to bite the bullet here and note it in the changelogs.